R/api.R

```diff
@@ -160,6 +170,10 @@ fetch_next <- function(track_id = "r", skip = FALSE,
     warning(sprintf("Not submitted: %s (%s)", x$name, x$language),
             call. = FALSE)
   }
+  
+  if (start_testing) {
+    start_testing(slug = slug, track_id = track_id)
```

Comment:
You need to use `x$slug` here, else in the case when `skip = TRUE`, the testing is applied to the skipped `slug`.